pom.xml

```diff
@@ -143,6 +143,23 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+			<dependency>
+				<groupId>org.hibernate</groupId>
+				<artifactId>hibernate-core</artifactId>
+				<version>5.2.12.Final</version>
```

Comment:
It maybe possible to extract this and use `<hibernate.version>` but since I was having so much trouble overriding, I hardcoded it everywhere and just left it when it worked.